submissions/7SergSmith7/Tiny JS World/index.js

```diff
@@ -0,0 +1,125 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Habitant {
+  constructor(species, name, gender, phrase, friends = [], legs, hands) {
+    this.species = species;
+    this.legs = legs;
+    this.hands = hands;
```

Comment:
Not every Habitant has hands.
The beuaty of OOP approach is that each super (parent) class defines and handles only shared properties, and each child class introduces and handles properties unique to that child class.
Imagine, you'd need add birds, worms etc. What super class would look like then?
Let's however stick to mammals.

Reply:
done